app/server.py

```diff
@@ -16,26 +18,25 @@
 from rasabot import RasaBotProcess, RasaBotTrainProcess
 from datetime import datetime, timedelta
 from models.models import Bot
+from models.base_models import DATABASE
 from decouple import config
 
+
 class BotManager():
-    '''
-    javascript client:
-    var ws = new WebSocket('ws://localhost:8888/ws');
-    ws.onmessage = (evt) => {
-        console.log(JSON.parse(evt.data));
-    }
-    var bot_message = {
-        question: 'Whos there?',
-        botId: '123456'
-    }
-    ws.send(JSON.stringify(bot_message))
-    '''
+    """
+    Bot mananger responsible to manager all bots in this instance
+    """
     _pool = {}
 
     def __init__(self):
 
-        self.redis = redis.ConnectionPool(host=config('BOTHUB_REDIS'), port=6379, db=2)
+        self.redis = redis.ConnectionPool(
+            host=config('BOTHUB_REDIS'),
+            port=config('BOTHUB_REDIS_PORT'),
+            db=config('BOTHUB_REDIS_DB')
+        )
+        self._set_instance_redis()
+        self._set_server_alive()
         self.start_garbage_collector()
 
     def _get_bot_data(self, bot_uuid):
```

Comment:
Would be good use `logging` lib from Python to print some logs about the process instead of `print` command.

Reply:
This has already been changed :D